### PR TITLE
Chatglmv2 attention mask

### PR DESCRIPTION
**Pull request by wtmlon**
### Description

统一 chatglm2 attention mask 处理

### CODE REVIEW

**Comment by lugimzzz** [comment]
attention_mask部分修改review了没有问题，注意保证llm/causallm目录下和llm/chatglm中prefix可跑通

**Review by lugimzzz — APPROVED** [review]
lgtm
